applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -91,15 +91,17 @@ void BaseSolidElement::InitializeSolutionStep( const ProcessInfo& rCurrentProces
         }
     }
     if (required) {
-        const SizeType number_of_nodes = GetGeometry().size();
-        const SizeType dimension = GetGeometry().WorkingSpaceDimension();
+        const auto &r_geom = GetGeometry();
```

Comment:
```suggestion
        const auto& r_geom = GetGeometry();
```
I like more like this, but as you want